pkg/service/repair/service.go

```diff
@@ -888,6 +917,18 @@ func (i *intensityHandler) SetParallel(ctx context.Context, parallel int) error
 	return nil
 }
 
+// SetSingleHostParallelism sets repair max parallelism per host value.
+func (i *intensityHandler) SetSingleHostParallelism(ctx context.Context, singleHostParallelism int) error {
+	if singleHostParallelism < defaultSingleHostParallelism {
```

Comment:
Shouldn't it be lesser than 0, not the default value?

Reply:
defaultSingleHostParallelism is const = 0

Reply:
@Michal-Leszczynski was right - the default should have been 1.
@karol-kokoszka I wonder what's the point of a review if you ignore completely relevant reviewers comments like here?

Reply:
@vladzcloudius it cannot be default to 1. The main reason for that is "We don't want to change the current behavior of the repair in the manager". By default, there is no limit on this parallelism. This change cannot be invasive for the current users.
We checked with the SCT tests, that setting this parallelism to 1 lowers the performance of the overall repair.

Reply:
But there IS limit for this parameter and it's documented and it's exactly `1`.
@karol-kokoszka I don't think it makes sense to discuss the PR any longer. 

We are going to take it with @tzach as soon as he returns.
Until then I suggest to not add any additional noise in the public repo.

Please, wait for an update from Tzach about this parameter in general and changes in repair APIs in particular.